packages/walletconnect/package.json

```diff
@@ -32,13 +32,17 @@
     "@celo/utils": "^2.2.0",
     "@celo/wallet-base": "^2.2.0",
     "@celo/wallet-remote": "^2.2.0",
-    "@walletconnect/client": "2.0.0-beta.21",
-    "@walletconnect/types": "2.0.0-beta.21",
-    "@walletconnect/utils": "2.0.0-beta.21",
+    "@walletconnect/auth-client": "^2.0.2",
+    "@walletconnect/sign-client": "^2.3.2",
+    "@walletconnect/types": "^2.3.2",
+    "@walletconnect/utils": "^2.3.2",
     "debug": "^4.3.3",
     "ethereumjs-util": "^7.1.3"
   },
   "engines": {
     "node": ">=10"
+  },
+  "devDependencies": {
+    "lokijs": "^1.5.12"
```

Comment:
whats that?

Reply:
that's a dependency that WC has on server side (so it was throwing without it in tests)